extensions/wikia/PhalanxII/hooks/PhalanxContentBlock.class.php

```diff
@@ -207,17 +207,17 @@ static public function editContent( $textbox, &$error_msg, $phalanxModel = null,
 	 *
 	 * hook
 	 */
-	static public function checkContent( $textbox, &$msg ) {
+	static public function checkContent( $textbox, &$msg, $displayBlock = false ) {
 		wfProfileIn( __METHOD__ );
 
 		$title = RequestContext::getMain()->getTitle();
 
 		$phalanxModel = new PhalanxContentModel( $title );
 
 		$ret = PhalanxContentBlock::editContent( $textbox, $msg, $phalanxModel );
-		
-		if ( $ret === false ) {
-			$msg = $phalanxModel->textBlock();
```

Comment:
And for clarity—`textBlock()` doesn't do any magic and should be rather called `getTextBlockMessageAndLog()`, because it looks like this:
```
public function textBlock() {
	$this->logBlock();
	return $this->block->text;
}
```